ambry-store/src/main/java/com.github.ambry.store/IndexSegment.java

```diff
@@ -244,11 +242,11 @@ Offset getEndOffset() {
   }
 
   /**
-   * Returns if this segment is mapped or not
-   * @return True, if the segment is readonly and mapped. False, otherwise
+   * Returns if this segment is immutable or not
+   * @return {@code true}, if the segment is sealed
```

Comment:
The words immutable and sealed are being used interchangeably throughout the class.  It would be good to settle on one, at least for the docs.  Also, a single @return comment is sufficient.

Reply:
+1

Reply:
Settled on "sealed"